src/db/schema.ts

```diff
@@ -73,13 +79,20 @@ export const account = pgTable("account", {
   refreshToken: text("refresh_token"),
   expiresAt: timestamp("expires_at", { withTimezone: true }),
   password: text("password"),
+  createdAt: timestamp("created_at").defaultNow().notNull(),
+  updatedAt: timestamp("updated_at").defaultNow().notNull(),
```

Comment:
Follow the other pattern.
```suggestion
  createdAt: timestamp("created_at", { withTimezone: true }).notNull().defaultNow(),
  updatedAt: timestamp("updated_at", { withTimezone: true })
    .notNull()
    .defaultNow()
    .$onUpdate(() => new Date()),
```

Reply:
No, it just slipped my mind. I thought i had changed it. i will change it promptly